v2/workloadapi/client_test.go

```diff
@@ -1 +1,210 @@
 package workloadapi
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/spiffe/go-spiffe/v2/bundle/jwtbundle"
+	"github.com/spiffe/go-spiffe/v2/proto/spiffe/workload"
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+	"github.com/spiffe/go-spiffe/v2/svid/jwtsvid"
+	"github.com/spiffe/go-spiffe/v2/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"gopkg.in/square/go-jose.v2"
+)
+
+type ClientTestSuite struct {
```

Comment:
How would you feel about not using the test suite functionality and instead just using regular test methods? Test suites tend to grow very complicated setup and teardown methods that aren't always applicable to every test which sometimes leads to unintended maintenance consequences. It's a lot easier to see exactly what preconditions a test needs if it has an explicit setup.

Reply:
No strong feelings about suites. I'm currently in the process of changing this.

Reply:
Thanks :heart: